frontend/src/pages/article/components/article-view/article-view.tsx

```diff
@@ -44,6 +46,13 @@ const ArticleView: React.FC<Properties> = ({ article }) => {
             iconClassName={styles.icon}
             onClick={onButtonClick}
           />
+          <ShareOnFacebookButton
+            title={title}
+            size="54"
```

Comment:
Could be moved to constant together with bgColor and iconColor.

Reply:
I will change the icon and delete these props. So there is no need